go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/giantswarm/appcatalog v0.4.1
 	github.com/giantswarm/backoff v0.2.0
 	github.com/giantswarm/exporterkit v0.2.1
-	github.com/giantswarm/helmclient/v4 v4.5.0
+	github.com/giantswarm/helmclient/v4 v4.5.1-0.20210430080600-653555481d8d
```

Comment:
Move it to v4.6.0 after merge & release new version from https://github.com/giantswarm/helmclient/pull/341